packages/solana-v1-contrib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tensor-hq/solana-v1-contrib",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Please bump this to 0.2.0 (semver says new functionality is a minor bump: https://semver.org/)